modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="cco-ccoctl-deleting-sts-resources_{context}"]
+= Deleting AWS resources with the Cloud Credential Operator utility
+
+After installing an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created. These resources are required only during changes that require elevated permissions, such as upgrades.
```

Comment:
```suggestion
After installing an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created.
```

All the resources created by ccoctl are necessary while the cluster is running. Some of those resources may need updating during upgrades, but the resources are pretty much always needed.